libs/editor/libs/editor-common/assets/ZSSRichTextEditor.js

```diff
@@ -1087,8 +1087,11 @@ ZSSEditor.markAllUploadingImagesAsFailed = function(message) {
 
     for(var i = 0; i < matches.size(); i++) {
         var mediaId = matches[i].getAttribute("data-wpid");
+        var videoId = matches[i].getAttribute("data-video_wpid");
         if (mediaId != null) {
```

Comment:
From the [getAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute) documentation:

> If the given attribute does not exist, the value returned will either be null or "" (the empty string);

Considering there are differences on Android Webviews, I wonder if we should check with `hasAttribute` first or test if the returned value is the empty string.